app/setup.py

```diff
@@ -491,3 +479,16 @@ def get_minimized_asset(filename):
         elif "js" in filename:
             filename = filename.replace(".js", ".min.js")
     return filename
+
+
+def get_locale():  # pylint: disable=unused-variable
+    session = get_session_store()
+
+    if session and (session_data := session.session_data):
+        return session_data.language_code
+
+    return None
+
+def get_timezone():  # pylint: disable=unused-variable
```

Comment:
The pylint disable statement still needed?

Reply:
Removed the pylint disable comments, Pylint test passed 👍